scripts/requirements.txt

```diff
@@ -1,2 +1,4 @@
 -r requirements-base.txt
 -r requirements-build.txt
+-r requirements-ci.txt
```

Comment:
@jaci-nordic removed it from requirements.txt and added to to the pip-compile command in compile-requirements.txt

Reply:
could you please fixup this commit: https://github.com/nrfconnect/sdk-nrf/pull/12193/commits/7851b12da6f944c7fbeb9d749f66f788f25fc2b5
to avoid adding the lines in the first place.

It's annoying when reviewing PRs or doing a blame later, to have lines added in one commit, which are then removed again in the next commit.

In this case, those lines:
https://github.com/nrfconnect/sdk-nrf/blob/7851b12da6f944c7fbeb9d749f66f788f25fc2b5/scripts/requirements.txt#L3-L4
are added in this commit: https://github.com/nrfconnect/sdk-nrf/pull/12193/commits/7851b12da6f944c7fbeb9d749f66f788f25fc2b5
and removed in this commit: https://github.com/nrfconnect/sdk-nrf/pull/12193/commits/0176f8f4850d169ea72017d23702480c6845179d